extensions/wikia/Flags/tasks/FlagsExtractTemplatesTask.php

```diff
@@ -0,0 +1,116 @@
+<?php
+namespace Flags;
+
+use Flags\FlagsExtractor;
+use Wikia\Tasks\Tasks\BaseTask;
+
+class FlagsExtractTemplatesTask extends BaseTask {
+
+	private
+		$actionsStatus = false;
```

Comment:
I was going to extract the add and update methods and make them call the same property. Leave it until refactoring?